spec/features/create_media_audio_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'random_word'
+include ActiveJob::TestHelper # have background jobs run synchronously
+
+RSpec.describe 'Create Media Audio objects', type: :feature do
+  let(:project_name) { "Media-audio-objects-#{RandomWord.nouns.next}" }
+  let(:bundle_dir) { Rails.root.join('spec/test_data/media_audio_test') }
+  let(:cocina_model_world_access) { instance_double(Cocina::Models::Access, access: 'world') }
+  let(:item) { instance_double(Cocina::Models::DRO, type: Cocina::Models::Vocab.media, access: cocina_model_world_access) }
+  let(:dsc_object_version) { instance_double(Dor::Services::Client::ObjectVersion, openable?: true) }
+  let(:dsc_object) { instance_double(Dor::Services::Client::Object, version: dsc_object_version, find: item) }
+
+  before do
+    # fake devise user
+    user = FactoryBot.create(:user)
```

Comment:
since specs include `FactoryBot::Syntax::Methods`, you can omit the `FactoryBot` here.
```suggestion
    user = create(:user)
```